website/docs/language/settings/terraform-cloud.mdx

```diff
@@ -29,7 +29,7 @@ terraform {
 }
 ```
 
-If you do not specify the `hostname`, it defaults to `app.terraform.io` for Terraform Cloud. For Terraform Enterprise installations, include the [hostname](/cli/cloud/settings#hostname) configuration argument.  
+If you do not specify the `hostname`, it defaults to `app.terraform.io` for Terraform Cloud. For Terraform Enterprise installations, include the [hostname](/cli/cloud/settings#hostname) configuration argument.
 
 You cannot use the CLI integration and a [state backend](/language/settings/backends) in the same configuration; they are mutually exclusive. A configuration can only provide one `cloud` block and the `cloud` block cannot refer to named values like input variables, locals, or data source attributes.
```

Comment:
```suggestion
You cannot use the CLI integration and a [state backend](/language/settings/backends) in the same configuration because they are mutually exclusive. A configuration can only provide one `cloud` block and the `cloud` block cannot refer to named values like input variables, locals, or data source attributes.
```

Suggest changing semicolon to "because"